### DISCUSSION TITLE
Best settings in g-helper for a Zephyrus g16 ( Ultra 9 and 4070 )

### DISCUSSION THREAD

**Discussion by Retr0-rep** [seerge/g-helper#3350] [Q&A]
I feel like my device tends to get unnecessarily hot under minimal load, but i have seen improvements after deleting AC, and i was hoping on getting better temps similar to reviews iv watched on yt for this model. I still do feel like my device is unnecessarily hot, so was hoping to get some recommendations for better performance and battery life according to the mode in my specific model in the different modes (as in the profiles for like turbo, silent and performance). Thank you in advance for any help provided!

**Comment by seerge** [comment]
@Retr0-rep hello,

Generally speaking default profiles (baked in BIOS) should be already good, as they are tuned for your specific device by the manufacturer.

If you want to tackle heat a bit, you can try to limit your CPU power in addition to everything.  Modern Intel or AMD CPUs are quite efficient, and usually for most of the graphics-intensive games setting CPU power limit to something like 30-40W is more than enough.

For battery life just make sure you use Silent and Eco (or Optimized) and set Battery Saver mode in windows tray menu.

**Reply by Retr0-rep** [reply]
Thank you very much!

**Reply by seerge** [reply]
Just a small remark. 

On some devices (including new G14 and G16) as soon as you apply custom power limits BIOS starts treating this mode as "custom / manual" (same as it happens when you do it in Armoury), so it can also change fan behavior. In that case if fan behavior would be a bit different from "default" - that's normal, so you may tweak it for your liking as well then.

**Reply by Retr0-rep** [reply]
Oh I see... once again thank you!
